_posts/blog/2017-05-15-how-to-structure-product-feedback.md

```diff
@@ -0,0 +1,42 @@
+Providing feedback on products is tricky. When done properly it can serve as a valuable tool that fosters positive relationships and improves the products we build. When done poorly it can crush spirits and misinform the decisions we make. Through research and practice I’ve established several steps I now use to structure the way I give feedback. 
+
+## Avoid Generalized Statements
```

Comment:
Because the blog title is heading level 1 (`h1`) and the post title is level 2 (`h2`), all your top level headings in your post should be `h3` instead of `h2`. This is just how the TMS site is structured, so that's something we have to do for all our posts (at least for now).